torchvision/models/video/resnet.py

```diff
@@ -189,16 +211,22 @@ def __init__(self):
 
 class VideoResNet(nn.Module):
 
-    def __init__(self, block, conv_makers, layers,
-                 stem, num_classes=400,
-                 zero_init_residual=False):
+    def __init__(
+        self,
+        block: Type[Union[BasicBlock, Bottleneck]],
+        conv_makers: List[Type[Union[Conv3DSimple, Conv3DNoTemporal, Conv2Plus1D]]],
+        layers: List[int],
+        stem: Type[nn.Module],
+        num_classes: int = 400,
+        zero_init_residual: bool = False,
+    ) -> None:
         """Generic resnet video generator.
 
         Args:
             block (nn.Module): resnet building block
             conv_makers (list(functions)): generator function for each layer
             layers (List[int]): number of blocks per layer
-            stem (nn.Module, optional): Resnet stem, if None, defaults to conv-bn-relu. Defaults to None.
+            stem (Type[nn.Module]): module class to instantiate the ResNet stem.
```

Comment:
```suggestion
            stem (Callable[..., nn.Module]): module class to instantiate the ResNet stem.
```

What you wrote is correct but it seems on Vision we use a [different pattern](https://github.com/pytorch/vision/search?q=%22Callable%5B...%2C+nn.Module%5D%22). Perhaps keep it for consistency reasons?